components/fetch-build-scan-data-cmdline-tool/src/main/java/com/gradle/enterprise/cli/FetchBuildValidationDataCommand.java

```diff
@@ -59,7 +59,7 @@ public Integer call() throws Exception {
         // Use System.err for logging since we're going to write out the CSV to System.out
         logger = new ConsoleLogger(System.err, colorScheme, debug);
 
-        ProxyServerConfigurator.configureProxyServers(proxySettingsFile, logger);
+        NetworkingConfigurator.configureNetworking(networkSettingsFile, logger);
```

Comment:
In the code `networking` is used, for the file name `network` is used. Would it make sense to align on the naming? If you think so, too, then I would likely go with `network`. Curious what you think.

Reply:
I renamed `NetworkingConfigurator` to `NetworkSettingsConfigurator`.  I went with "network settings" because I didn't want to imply that we're configuring the whole network for a host.  Although that still exists.

We could consider renaming everything to "HTTP settings" (`HttpConfigurator`, `http.settings`, etc), which might be slightly more accurate (all these settings only deal with configuring the HTTP).